american-journal-of-climate-change.csl

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>American Journal of Climate Change</title>
+    <title-short>AJCC</title-short>
+    <id>http://www.zotero.org/styles/american-journal-of-climate-change</id>
+	<link href="http://www.zotero.org/styles/american-journal-of-climate-change" rel="self"/>
+	<issn>2167-9495</issn>
+      <eissn>2167-9509</eissn>
+	<link href="http://www.scirp.org/journal/ajcc/" rel="documentation"/>
```

Comment:
@arunbte, if you know which style you used as a template, please add a "template" link. See https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link